waspc/data/Generator/templates/server/src/jobs/core/simpleJob.js

```diff
@@ -2,14 +2,14 @@ import { sleep } from '../../utils.js'
 import { Job } from './Job.js'
 import { SubmittedJob } from './SubmittedJob.js'
 
-export const PASSTHROUGH_EXECUTOR_NAME = Symbol('Passthrough')
+export const SIMPLE_EXECUTOR_NAME = Symbol('Simple')
 
 /**
  * A simple job mainly intended for testing. It will not submit work to any
  * job executor, but instead will simply invoke the underlying perform function.
- * It is dependency-free, however.
+ * It does not support `schedule`. It is dependency-free, however.
```

Comment:
What does it mean it is dependency-free? Sounds like it doesn't need any npm dependencies, but I mean you want to say it doesn't need any additional infrastructure? Not sure hm. Maybe it says exactly what you meant to say.

Reply:
Good call, I can update it to be does not require any additional infrastructure or npm dependencies. 👍🏻